.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "singleQuote": true
```

Comment:
A personal preferences in this case, just because it's faster to write `'` than `"`. :smile: 
I'll remove it and use the default anyway.

Reply:
My original approach with YAML was to use single quotes wherever possible (because they are the most restrictive, and thus less likely to result in unintended behavior), then double quotes only where single quotes won't do.

But I'm noticing that the common YAML style seems to be to just the opposite: no quotes wherever possible, then use double quotes where no quotes  won't work, like this default output from `arduino-cli config init`:
```yaml
board_manager:
  additional_urls: []
daemon:
  port: "50051"
directories:
  data: /home/per/.arduino15
  downloads: /home/per/.arduino15/staging
  user: /home/per/Arduino
logging:
  file: ""
  format: text
  level: info
telemetry:
  addr: :9090
  enabled: true
```
I figured that's the most "official" YAML Arduino has, so I've been trying to move in that direction when I have no existing guidance on a YAML document.